src/Identity/samples/IdentitySample.Mvc/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -15,6 +15,7 @@ partial class ApplicationDbContextModelSnapshot : ModelSnapshot
     {
         protected override void BuildModel(ModelBuilder modelBuilder)
         {
+#pragma warning disable 612, 618
```

Comment:
nit: we generally do:
```csharp
#pragma warning disable CS0618 // Type or member is obsolete
// ...
#pragma warning restore CS0618 // Type or member is obsolete
```

Reply:
I aligned it to what EF Core prints in migration files & what is in other migration files in the repo already.